### ISSUE TITLE
New metrics for Cancer Screening Survey (December release)

### ISSUE THREAD

**Issue by robertsamm** [Analyticsphere/metricsReportsRequests#179]
The Cancer Screening Survey will be released in prod in the December release to both prospective and retrospective participants. Eligibility requirements to receive the survey are verified and not withdrawn with a verification date of 9-months (270 days) or longer. Eligible participants will have a ‘not started/started/submitted’ flag while not-eligible or not-yet-eligible participants will have a survey flag of ‘null’. Acronym for survey can be ‘CSS’, where needed. 
See [requirements](https://nih.app.box.com/file/1649462024834) on Box for more details. 
Related github issue here: https://github.com/episphere/connect/issues/1145

In anticipation of the Cancer Screening Survey release, please add the following metrics to the Weekly Survey Module Metrics report.

General Updates

- [x] Add Cancer Screening Survey as a new ‘section’ and add to the TOC. This can be placed between the ‘Connect Experience Survey’ and the ‘Survey Metrics by Language Completed’

Cancer Screening Survey Section – new metrics

- [x] Add similar metrics to the Weekly Module Metric's Report tables 4.1-4.5

Updated Metrics

- [x] Add CSS survey to tables 5.3-5.3b on survey completion by survey language

**Comment by KELSEYDOWLING7** [comment]
@robertsamm One question on this. Table 4.4 for the Connect Experience Survey has a set trigger time. It looks like these participants will get notifications in waves. So should I set the trigger time as their first notification date?

**Comment by brotzmanmj** [comment]
@KELSEYDOWLING7  yes that would be reasonable

**Comment by KELSEYDOWLING7** [comment]
Perfect. Then this code is done and ready to be tested once we have stage data

**Comment by KELSEYDOWLING7** [comment]
With dev data:

![image](https://github.com/user-attachments/assets/fbaf30b7-3443-4a52-8af2-366b30db19e4)
![image](https://github.com/user-attachments/assets/db48277a-8559-418f-9417-ab8ab3dc07e1)
![image](https://github.com/user-attachments/assets/22442fab-3c47-4983-b02a-17c707bcb518)
![image](https://github.com/user-attachments/assets/5f40836f-c625-48d3-9499-99a130385a04)
![image](https://github.com/user-attachments/assets/10bda0d8-08c3-4e12-8112-5fffb577fff4)


![image](https://github.com/user-attachments/assets/b82884d4-fff3-45b9-bf18-e2d206c38683)
![image](https://github.com/user-attachments/assets/90002950-baf6-4108-a21e-0d3a4bc8c7b7)
![image](https://github.com/user-attachments/assets/407f3111-3ab3-4177-8082-592d37c07fae)

**Comment by brotzmanmj** [comment]
This looks good to me, thanks! @robertsamm and @kmazzilli  all ok or any changes?

**Comment by kmazzilli** [comment]
Looks good to me as well!

**Comment by KELSEYDOWLING7** [comment]
Will be put in the report after the January prod push

**Comment by brotzmanmj** [comment]
Hi @KELSEYDOWLING7 as you know, we released the cancer screening survey in production yesterday so we are ready for these metrics to start to populate. Will this be in the metrics report in prod when it runs on Monday?

Also, there is one table we missed that wasn't requested but we need to add. We'd like to know CSH survey completion rates based on time since verification, essentially the same idea table as Table 3.10 that we did for PROMIS. Can you add it but with title: CSH Survey Completion from Time of Verification to Date of First CSH Notification and the intervals should be More than 18 months, 12-18 months, and 9-12 months.

**Comment by KELSEYDOWLING7** [comment]
Yes that can be added. I've been working with Jake to get the cancer screening history table flattened and the new CSH variables in the flattened participants table. I am OOO on Thursday and Friday so we can't do an automation update, but I can run these reports manually on Monday to include the changes

**Comment by brotzmanmj** [comment]
That would be great, thank you!

**Comment by KELSEYDOWLING7** [comment]
![image](https://github.com/user-attachments/assets/8e027f2e-16a7-4162-bcc6-1c4efeddd302)
![image](https://github.com/user-attachments/assets/abc337ca-eb22-4ed5-802a-b932f6dbdb34)
![image](https://github.com/user-attachments/assets/25a68407-9bc1-418d-8abd-30128bae71fc)
![image](https://github.com/user-attachments/assets/0673cefa-95ed-4193-9b01-dcc8f1887c2f)
![image](https://github.com/user-attachments/assets/e183fbbc-7f2c-4557-bb6b-77cd64897c2a)

**Comment by brotzmanmj** [comment]
Looks great, thank you!
